packages/x-charts-pro/src/context/ZoomProvider/defaultizeZoom.ts

```diff
@@ -0,0 +1,94 @@
+import { AxisConfig, ScaleName, ChartsXAxisProps } from '@mui/x-charts';
+import { isDefined } from '@mui/x-charts/internals';
+import { DefaultizedZoomOptions, ZoomOptions } from './Zoom.types';
+
+const defaultZoomOptions = {
+  start: 0,
+  end: 100,
+  step: 5,
+  minSpan: 10,
+  maxSpan: 100,
+  panning: true,
+};
+
+export const defaultizeZoom = (
+  axis: Pick<AxisConfig<ScaleName, any, ChartsXAxisProps>, 'id' | 'zoom'>[] | undefined,
+  axisName: 'x' | 'y',
```

Comment:
When seeing this attribute I was wondering which kind of special treatment you're doing for x/y axes.

Could be clearer to do  `defaultizeZoom( ... ).map((option) => ({ ...option, axis: 'x'}) )`. But that's more a question of taste

Maybe also rename it. `axis` is already used a lot. Maybe `axisDirection`

Reply:
Mapping is unnecessary I think, but I like the naming. It is necessary for the zoom config, as for the X axis I need to get where the mouse is horizontally, while for the Y axis it needs to be vertically.